src/index.js

```diff
@@ -1,11 +1,37 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { createStore } from 'redux';
 import './index.css';
-import App from './App';
+import { Provider } from 'react-redux';
+import rootReducer from './reducers/index';
+import App from './components/App';
+// import createBook from './actions';
```

Comment:
- [ ] Comments: It’s better to delete commented piece of codes. If you need to review a previous version of your code, use git history.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png)